modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/manually-creating-alibaba-ram.adoc
+
+:_content-type: PROCEDURE
+[id="manually-creating-alibaba-ram-user_{context}"]
+= Creating the required RAM user
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+When you configure the Resource Access Management (RAM) user, be sure to consider the following requirements:
+
+* Attach the `AdministratorAccess` and `AliyunBSSFullAccess` policies to a RAM user to ensure that the account has sufficient permission to create the {product-title} cluster. This policy grants permissions to manage all Alibaba Cloud resources. The user must have the following permissions:
++
+** ram:CreatePolicy
```

Comment:
I believe this set of actions is not reflecting what the current installer needs. I can't even create the 
- user's permission
```
$ aliyun ram ListPoliciesForUser --UserName $install_user
{
	"Policies": {
		"Policy": [
			{
				"AttachDate": "2022-02-09T18:41:06Z",
				"DefaultVersion": "v1",
				"Description": "",
				"PolicyName": "pr-doc-review",
				"PolicyType": "Custom"
			}
		]
	}
}
```

- policy document
`$ aliyun ram GetPolicy --PolicyName pr-doc-review --PolicyType Custom |jq -r .DefaultPolicyVersion.PolicyDocument  |jq .Statement[] `
```json
{
  "Effect": "Allow",
  "Action": [
    "ram:CreatePolicy",
    "ram:GetPolicy",
    "ram:CreatePolicyVersion",
    "ram:DeletePolicy",
    "ram:DetachPolicyFromUser",
    "ram:ListPoliciesForUser",
    "ram:AttachPolicyToUser",
    "ram:CreateUser",
    "ram:GetUser",
    "ram:DeleteUser",
    "ram:CreateAccessKey",
    "ram:ListAccessKeys",
    "ram:DeleteAccessKey",
    "bss:CreateInstance",
    "bss:Query",
    "bss:ModifyInstance"
  ],
  "Resource": "*"
}

```
- installation
```
$ ./openshift-install create install-config --dir $INSTALL_DIR
? SSH Public Key /home/mtulio/.ssh/id_rsa.pub
? Platform alibabacloud
? Region us-east-1
FATAL failed to fetch Install Config: failed to fetch dependency of "Install Config": failed to generate asset "Base Domain": SDK.ServerError 
FATAL ErrorCode: Forbidden.RAM                     
FATAL Recommend: https://error-center.aliyun.com/status/search?Keyword=Forbidden.RAM&source=PopGw 
FATAL RequestId: 002112A4-97C9-51E5-88BC-399B7182011E 
FATAL Message: User not authorized to operate on the specified resource, or this API doesn't support RAM. 

```